x/programs/rust/README.md

```diff
@@ -20,10 +20,14 @@ cargo build --target wasm32-unknown-unknown --target-dir $CARGO_TARGET_DIR --rel
 
 ## Debugging
 
-While developing programs you can optionally compile the program with wasi
-support. This provides for example logging from your program. The compile Wasm
-will be automatically supported by the VM simulator or if you explicitly set
-WithEnableTestingOnlyMode(true) for the runtime `Config`.
+While developing programs you can optionally compile the program with `Wasi`
+support. This allows, for example, logging from your program. The compiled
+`Wasm` will be automatically supported by the VM simulator, or if you explicitly
+set `WithEnableTestingOnlyMode` for the runtime `Config`. 
+
+**NOTE**: Once testing is complete, remove all print statements from your
```

Comment:
If you keep them around, will it error or just skip it?

Reply:
there is no error if you recompile, if you don't it will error.

Reply:
Decided to leave the script and clarify the docs